QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/rapidpass-and-qr-ticket-handler.js

```
@@ -0,0 +1,159 @@
+        doc.setFont('helvetica', 'bold');
+        doc.setTextColor('#006a4e');
+        doc.text('Dhaka Metro Rail', 50, 15, { align: 'center' });
+
+        // Add QR code to PDF
+        doc.addImage(ticketData.qrCodeImage, 'PNG', 10, 20, 80, 80);
+
+        // Add ticket information
+        doc.setFontSize(11);
+        doc.setFont('helvetica', 'normal');
+        doc.setTextColor('#333333');
+        doc.text('From: ' + ticketData.originStationName, 10, 112);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)